src/components/Pages/OfficialDocuments/OfficialDocumentPage.js

```diff
@@ -18,15 +19,28 @@ const OfficialDocumentPage = ({ officialDocuments, intl }) => {
   const [ pageNumber, setPageNumber ] = useState(getHash())
   const shownPages = buildPagination()
   const page = pages[pageNumber]
+  let domWindow // this localized variable allows us to pass the DOM 'window' between methods without drawing errors on node js builds in react.
 
   useEffect(() => {
+    domWindow = window
     window.onpopstate = function(event) {
       setPageNumber(getHash())
-      window.requestAnimationFrame( ()=> window.scroll(0,0) )
     }
     window.location.hash = pageNumber+1
   })
 
+  function changePage(newPage) {
+    if (newPage >= 0 && newPage <= pages.length - 1) {
+      scrollTransition({
+        scrollDuration: 0.5, // Scroll effect duration, regardless of height, in seconds
+        fadeDelay: 0.15, // for both fade in & out. so 2x times value here for full transition.
+        element: domWindow,
+        fadeElement: officialDocumentsPage,
+        callback:()=>{ setPageNumber(newPage) } // NOTE: callback will fire after fade OUT and BEFORE fade IN.
+      })
```

Comment:
Here is where the scroll transition helper method is called and the arguments passed!
- NOTE: callback will fire after fade OUT but BEFORE fade IN.